### ISSUE TITLE
Running Rekono from Docker issue - Nginx container keeps Failing

### ISSUE THREAD

**Issue by phoenixml** [pablosnt/rekono#231]
**Describe your problem, your doubts or whatever you need**
Hi There, 

I pulled the latest commits, once I finish the build successfully:
docker-compose build

I started the containers:
docker-compose up -d --scale executions-worker=5

I am seeing that nginx keeps failing after debugging the issue I can see the below is occuring:
....................++++
nginx                | writing new private key to '/etc/nginx/tls/privatekey.key'
nginx                | req: Can't open "/etc/nginx/tls/privatekey.key" for writing, Permission denied
nginx exited with code 1

Therefore not able to access rekono app from the browser  any idea how to solve this issue, thank you in advance appreciated.

**Comment by pablosnt** [comment]
Hello,

The problem is that Docker service can't access the private key used to configure the TLS communication. This key is generated automatically and stored in the directory `/etc/nginx/tls/` which is mapped to `docker/nginx/tls/` directory in your host.

Usually this problem happens if you clone the repository as root and execute Docker as  a regular user, so the container can't access the files in `docker/nginx/tls/`.

In conclusion, you need to give the proper permissions to your `docker/nginx/tls/` directory in your host. If you have doubts, about which permissions should you provide, just try `chmod -R 777 docker/nginx/tls` from the project root directory. Let me know if this solves your problem, I'm sorry for the inconvenience

**Comment by phoenixml** [comment]
Hello yes you are right, I missed this my bad. Ok I will fix it and try it thank you so much for your help. Appreciated 🙏🏻🙏🏻

**Comment by pablosnt** [comment]
You are welcome! I close the issue, please reopen it if you need anything else
